examples/sveltekit-pwa/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+	"compilerOptions": {
+		"moduleResolution": "node",
+		"module": "es2020",
+		"lib": ["es2020", "DOM", "WebWorker"],
+		"target": "es2020",
+		/**
+			svelte-preprocess cannot figure out whether you have a value or a type, so tell TypeScript
+			to enforce using \`import type\` instead of \`import\` for Types.
+			*/
```

Comment:
```suggestion
		/**
		 * svelte-preprocess cannot figure out whether you have a value or a type, so tell TypeScript
		 * to enforce using \`import type\` instead of \`import\` for Types.
		*/
```